proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -295,8 +296,11 @@ message GetRepositorySettingsResponse {
 message UpdateRepositorySettingsByNameRequest {
   string owner_name = 1;
   string repository_name = 2;
-  // Changes are only made to specified values. Unspecified ones are left unchanged.
   Visibility visibility = 3;
+  // Changes are only made to values specified in field_mask.
+  google.protobuf.FieldMask field_mask = 4;
```

Comment:
`visibility` is not affected by field mask, so that the existing code won't break